gnomad_qc/v3/annotations/generate_qc_annotations.py

```diff
@@ -101,32 +142,59 @@ def split_info() -> hl.Table:
     # Create split version
     info_ht = hl.split_multi(info_ht)
 
-    # Index AS annotations
     info_ht = info_ht.annotate(
         info=info_ht.info.annotate(
-            **{f: info_ht.info[f][info_ht.a_index - 1] for f in info_ht.info if f.startswith("AC") or (f.startswith("AS_") and not f == 'AS_SB_TABLE')},
-            AS_SB_TABLE=info_ht.info.AS_SB_TABLE[0].extend(info_ht.info.AS_SB_TABLE[info_ht.a_index])
+            **split_info_annotation(info_ht.info, info_ht.a_index),
         ),
-        lowqual=info_ht.lowqual[info_ht.a_index - 1]
+        AS_lowqual=split_lowqual_annotation(info_ht.AS_lowqual, info_ht.a_index),
     )
     return info_ht
 
 
-def generate_ac(mt: hl.MatrixTable, fam_file: str) -> hl.Table:
+def generate_allele_data(mt: hl.MatrixTable) -> hl.Table:
+    """
+    Writes bi-allelic sites MT with the following annotations:
+     - allele_data (nonsplit_alleles, has_star, variant_type, and n_alt_alleles)
+    :param MatrixTable mt: Full unsplit MT
+    :return: Table with allele data annotations
+    :rtype: Table
+    """
+    ht = mt.rows().select()
+    allele_data = hl.struct(
+        nonsplit_alleles=ht.alleles, has_star=hl.any(lambda a: a == "*", ht.alleles)
+    )
+    ht = ht.annotate(allele_data=allele_data.annotate(**add_variant_type(ht.alleles)))
+
+    ht = hl.split_multi_hts(ht)
+    ht = ht.filter(hl.len(ht.alleles) > 1)
+    allele_type = (
```

Comment:
Comment from Laurent's PR on variant QC data

Reply:
I think the distinction is that `variant_type` is a variant-level annotation, and here, `was_mixed` is being annotated at the allele level. I don't know if that's helpful for other types of analyses we'd want to do, but I agree that for the purposes of RF modeling, we likely only need one representation of this information, and I think it's fine to keep it at the variant level.